### PR TITLE
Fixes small DevTest bugs

### PR DESCRIPTION
**Pull request by Laboredih123**
[BUG][MAPPING]
## About the PR 
Fixes some missing things like igniter switch.

## Why's this needed? 
bug bad, fix bug good

### CODE REVIEW

File: code/map.dm [hunk 1]
```diff
@@ -46,7 +46,8 @@ var/global/list/mapNames = list(
 	//"Icarus" =			list("id" = "ICARUS",		"settings" = "icarus",			"playerPickable" = FALSE),
 	//"Gehenna" =			list("id" = "GEHENNA",		"settings" = "gehenna",			"playerPickable" = FALSE),
 	"blank" =				list("id" = "BLANK",		"settings" = "", 				"playerPickable" = FALSE),
-	"blank_underwater" =	list("id" = "BLANK_UNDERWATER", "settings" = "", 			"playerPickable" = FALSE)
+	"blank_underwater" =	list("id" = "BLANK_UNDERWATER", "settings" = "", 			"playerPickable" = FALSE),
+	"Super Epic DevTest" =	list("id" = "DEVTEST",		"settings" = "devtest",			"playerPickable" = FALSE,	"MaxPlayersAllowed" = 69)
```
Comment (Azrun) [inline code/map.dm:50]:
```suggestion
	"DevTest" =	list("id" = "DEVTEST",		"settings" = "devtest",			"playerPickable" = FALSE,	"MaxPlayersAllowed" = 69)
```

File: code/map.dm [hunk 2]
```diff
@@ -1102,6 +1103,45 @@ var/global/list/mapNames = list(
...
+
+	escape_dir = EAST
+
+	merchant_left_centcom = /area/shuttle/merchant_shuttle/left_centcom/cogmap
+	merchant_left_station = /area/shuttle/merchant_shuttle/left_station/cogmap
+	merchant_right_centcom = /area/shuttle/merchant_shuttle/right_centcom/cogmap
+	merchant_right_station = /area/shuttle/merchant_shuttle/right_station/cogmap
+
+	valid_nuke_targets = list("super awesome area" = list(/area/devzone),
+		"another super test area" = list(/area/space))
```
Comment (ZeWaka) [inline code/map.dm:1142]: no

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/map.dm (hunk 2 above)
Comment (Laboredih123) [reply code/map.dm:1142]: ok

Comment (ZeWaka) [reply code/map.dm]: You seem to misunderstand. This is invalid code

**Pull request merged**